hazelcast/include/hazelcast/client/serialization/pimpl/compact.h

```diff
@@ -1522,6 +1529,29 @@ struct HAZELCAST_API field_operations
     static field_kind_based_operations get(enum field_kind field_kind);
 };
 
+namespace rabin_finger_print
+{
+
+/**
+ * We use uint64_t for computation to match the behaviour of >>> operator
+ * on java. We use >> instead.
+ */
+constexpr uint64_t INIT = 0xc15d213aa4d7a795L;
+
```

Comment:
Please put the documentation of each method similar to Java side doc.

Reply:
Fixed.